bundles/binding/org.openhab.binding.plex/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="plex"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>Plex Binding</name>
+    <description>The Plex Binding connects to a Plex Media Server to track the state of multiple clients.</description>
+    <author>Jeroen Idserda</author>
+
+    <service-id>org.openhab.plex</service-id>
+
+    <config-description>
+        <parameter name="host" type="text" required="true">
+            <context>network-address</context>
+            <label>Host</label>
+            <description>IP address of the Plex server</description>
```

Comment:
Could this be changed to support hostnames like your recent change?  (And that would mean the `<context>` element would no longer apply.)